packages/contracts/contracts/minter-suite/Minters/MinterSetPricePolyptychERC20V5.sol

```diff
@@ -902,12 +922,14 @@ contract MinterSetPricePolyptychERC20V5 is
                 tokenHashSeed: targetHashSeed
             });
 
-            uint256 newTokenId = (projectId * ONE_MILLION) + invocations;
-            PolyptychLib.setPolyptychHashSeed({
-                coreContract: coreContract,
-                tokenId: newTokenId, // new token ID
-                hashSeed: targetHashSeed
-            });
+            {
```

Comment:
nit: usually toss in an @dev comment about why block-scoping is used here

Reply:
updated in [41e4959](https://github.com/ArtBlocks/artblocks-contracts/pull/1141/commits/41e4959305fcb226e09b0aae86cc6c0b5860582f)